src/get_trigger.ts

```diff
@@ -35,18 +36,21 @@ const readJSONFile = async (path: string) => {
   }
 };
 
-const readTSFile = async (path: string) => {
-  try {
```

Comment:
Is there a reason we're dropping the `try/catch` block from here?

Reply:
Yeah, the `catch` doesn't do anything, just throws the error again. So it seems it's a no-op try/catch?

Reply:
No reason! I'll remove that one too, must have missed it.